src/storage/copier/rel_copier.cpp

```diff
@@ -23,6 +23,16 @@ RelCopier::RelCopier(CopyDescription& copyDescription, std::string outputDirecto
     initializePkIndexes(relTableSchema->dstTableID, *bufferManager);
 }
 
+std::vector<common::property_id_t> RelCopier::getAllPropertyIdxes() {
+    std::vector<common::property_id_t> propertyIdxes;
```

Comment:
bro! why is a vector of id is named as property**Idxes** instead of property**Ids**. We need to fix this naming across this PR.